tools/SourceKit/tools/sourcekitd/lib/API/CMakeLists.txt

```diff
@@ -16,5 +16,8 @@ endif()
 
 add_sourcekit_library(sourcekitdAPI
   ${sourcekitdAPI_sources}
-  DEPENDS SourceKitSupport
+  DEPENDS
+      SourceKitSupport SourceKitSwiftLang
+      # Clang dependencies, required by SourceKitSwiftLang
+      clangFormat clangToolingCore clangDriver
```

Comment:
Can you elaborate on the problem? Including LLVMExports.cmake and ClangTargets.cmake should setup all the transitive dependencies correctly.